dev/examples/reduceMotion.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react"
+import { motion, useReducedMotion } from "@framer"
+
+const style = {
+    width: 100,
+    height: 100,
+    background: "red",
+    opacity: 1,
+}
+
+export const App = () => {
+    const [isVisible, setIsVisible] = React.useState(false)
+    const isReducedMotion = useReducedMotion()
```

Comment:
Suggestion: use `shouldReduceMotion` as the result here as it reads better throughout the code an implies it's on the caller to do something with the value. `isReducedMotion` sounds like you're checking if motion has been suppressed somehow by the library.